nbdev/export.py

```diff
@@ -219,7 +223,7 @@ def _f(gps):
 
 # Cell
 def extra_add(code):
-    "Catch adds to `__all__` required by a cell with `_all_=`"
+    "Catch adds to `__all__` required by a cell with `%nbdev_add2all`"
```

Comment:
Mention both:
"Catch adds to `__all__` required by a cell with `_all_=` or `%nbdev_add2all`."

Reply:
done